packages/ra-core/src/CoreAdminRouter.js

```diff
@@ -93,15 +99,27 @@ export class CoreAdminRouter extends Component {
             theme,
             title,
         } = this.props;
-        const { children: childrenFromState } = this.state;
 
-        let childrenToRender =
-            typeof children === 'function' ? childrenFromState : children;
+        if (typeof children !== 'function' && !children) {
+            return (
+                <div style={welcomeStyles}>
+                    React-admin is properly configured.<br />
+                    Now you can add a first &lt;Resource&gt; as child of
+                    &lt;Admin&gt;.
+                </div>
```

Comment:
Why not also add a code snipper ? Great welcome message though :+1:

Reply:
Because the code for that message is bundled to production, even though it's only useful in development. I don't want to add too much weight to the final production bundle.